app.js

```diff
@@ -1,4 +1,4 @@
-const { getAllTopics, getAllArticlesWithCommentCount, getSpecificArticleById, getSpecificCommentsByArticleId} = require('./controllers')
+const { getAllTopics, getAllArticlesWithCommentCount, getSpecificArticleById, getSpecificCommentsByArticleId, postCommentByArticleId} = require('./controllers')
```

Comment:
Awesome to keep you've kept with the same naming convention in your controllers, It makes code super easy to read!